### DISCUSSION TITLE
Calendar Option - map field to relationship

### DISCUSSION THREAD

**Discussion by prodixx** [Laravel-Backpack/community-forum#900] [Q&A (Help)] [answered]
Hello!

Maybe this is a stupid question, but .. How can i map title to a relationship entry? I have EventCrud with calendar option and for event title i want to show customer name witch is CustomerCrud.

Also, the demo is not online, maybe you should specify or remove this info in Calendar Option details page.

Thanks

**Comment by pxpm** [comment, marked as answer]
Hey @prodixx hope you are doing well 🙏 

You can use an Accessor in your EventModel. See an example here: https://github.com/Laravel-Backpack/CRUD/issues/5461#issuecomment-1988020152

PS: make sure you eager-load the customer in the event to avoid lazy loading in the accessor. 

Calendar PR to demo is about to get merged too: https://github.com/Laravel-Backpack/demo/pull/589 thanks for the heads up 🙏 

Cheers
